src/main.rs

```diff
@@ -926,8 +964,17 @@ async fn run() -> Result<(), Error> {
                 Err(err) => event!(Level::ERROR, err = %err, "Pod reconciliation error."),
             }
         });
+
     let eip_controller = Controller::new(eip_api, ListParams::default())
         .run(reconcile_eip, on_error, context)
+        .then(|rr| async {
+            if rr.is_ok() {
+                if let Err(e) = report_eip_quota_status(&ec3_client, &quota_client).await {
```

Comment:
I don't have a good sense for how frequently the reconciler reconciles, but does this possibly hammer the quota endpoint? Does it need to be throttled?